src/main/java/redis/clients/jedis/BinaryJedisCluster.java

```diff
@@ -2411,6 +2410,11 @@ public Long execute(Jedis connection) {
     }.runBinary(key);
   }
 
+  @Override
+  public String unwatch() {
+    throw new UnsupportedOperationException();
```

Comment:
Should we put a message here?

```suggestion
    throw new UnsupportedOperationException("UNWATCH is not supported in cluster-mode");
```

Reply:
@mina-asham We absolutely should. This is somehow missed `TODO`.

Reply:
@mina-asham We absolutely should. This is somehow missed `TODO` :)